custom-config-server-client/src/main/resources/META-INF/spring.factories

```diff
@@ -1 +1 @@
-org.springframework.cloud.bootstrap.BootstrapConfiguration = com.microsoft.sample.customConfigServerClient.CustomConfigServiceBootstrapConfiguration
+org.springframework.boot.BootstrapRegistryInitializer=com.microsoft.sample.customConfigServerClient.CustomBootstrapRegistryInitializer
```

Comment:
nit: please add empty line at the end of file

Reply:
updated